content/sensu-go/5.17/installation/install-sensu.md

```diff
@@ -144,19 +144,56 @@ For a complete list of configuration options, see the [backend reference][6].
 
 ### 3. Initialize
 
-_**NOTE**: If you are using Docker, skip this step and continue with [4. Open the web UI][36]. The `sensu-backend init` command is not implemented for Docker._
-
 **With the backend running**, run `sensu-backend init` to set up your Sensu administrator username and password.
 In this initialization step, you only need to set environment variables with a username and password string &mdash; no need for role-based access control (RBAC).
 
-Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use:
+Replace `YOUR_USERNAME` and `YOUR_PASSWORD` with the username and password you want to use.
 
-{{< highlight shell >}}
+{{< language-toggle >}}
+
+{{< highlight Docker >}}
+docker run \
+-e SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME \
+-e SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD \
+--restart always \
+--name sensu \
+sensu/sensu:latest \
+sensu-backend start \
+sensu-backend init
+{{< /highlight >}}
+
+{{< highlight "Docker Compose" >}}
+---
+version: "3"
+services:
+  sensu-backend:
+    image: sensu/sensu:latest
+    ports:
+    - 3000:3000
+    - 8080:8080
+    - 8081:8081
+    volumes:
+    - "sensu-backend-data:/var/lib/sensu/etcd"
+    command: "sensu-backend-init -e SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME -e SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD"
+volumes:
+  sensu-backend-data:
+    driver: local
+{{< /highlight >}}
+
+{{< highlight "Ubuntu/Debian" >}}
 export SENSU_BACKEND_CLUSTER_ADMIN_USERNAME=YOUR_USERNAME
 export SENSU_BACKEND_CLUSTER_ADMIN_PASSWORD=YOUR_PASSWORD
 sensu-backend init
```

Comment:
Change `sensu-backend init` to `docker-compose up`. The instructions are the same for Ubuntu & RHEL; can probably get rid of the platform highlight here.

Reply:
I'm sorry, I think I'm misunderstanding this one -- These instructions are already in the doc for Ubuntu/Debian (see the existing [step 3](https://docs.sensu.io/sensu-go/latest/installation/install-sensu/#3-initialize)).
Because we're adding Docker instructions now, I added the tab and platform format we use in the docs in step 3. So it will appear on an Ubuntu/Debian tab like we have in [step 2](https://docs.sensu.io/sensu-go/latest/installation/install-sensu/#2-configure-and-start). They are the same as for RHEL/CentOS, but we use a platform switcher that separates the two.

Should we have docker instructions in the Ubuntu/Debian tab? Or do you mean we should add `docker-compose-up` at the end of the Docker Compose code?

Reply:
My bad - you can ignore this entire suggestion.